rfcs/main_branch.md

```diff
@@ -0,0 +1,28 @@
+# RFC #80: Rename `master` branch to `main` in all repositories
+
+## Summary
+
+Rename `master` branch to `main` in all repositories in the [web-platform-tests org](https://github.com/web-platform-tests).
+
+## Details
+
+These are our existing repos, excluding archived and private ones:
+- https://github.com/web-platform-tests/data-migration
+- https://github.com/web-platform-tests/editor
+- https://github.com/web-platform-tests/rfcs
+- https://github.com/web-platform-tests/wpt
+- https://github.com/web-platform-tests/wpt-empty
+- https://github.com/web-platform-tests/wpt-metadata
+- https://github.com/web-platform-tests/wpt-pr-bot
+- https://github.com/web-platform-tests/wpt.fyi
+- https://github.com/web-platform-tests/wpt.live
+
+TODO: which external repos hardcode the name of the default branch in any WPT repo?
```

Comment:
Probably many bots that interact with wpt e.g. the gecko sync bot definitely has some hardcoded branch names.

Reply:
Yep, it's all over Chromium's 2-way sync too:
https://source.chromium.org/chromium/chromium/src/+/main:third_party/blink/tools/blinkpy/w3c/local_wpt.py;l=39;drc=d9363a8ca4d0967ce169cad2c9d25534a1e0c25a

Reply:
I wasn't planning to flesh this out today, but I think what I'd propose here to make the transition work is to have both `main` and `master` branch updated and kept in sync for a while, so that the migration in other repos can happen at a leisurely pace and not strongly coordinated with the default branch change.

The alternative of having other repos be able to handle both branch names and later clean it up seems riskier and very un-fun.